Cargo.toml

```diff
@@ -16,10 +16,9 @@ travis-ci = { repository = "gimli-rs/addr2line" }
 
 [dependencies]
 gimli = { version = "0.20", default-features = false, features = ["read"] }
-fallible-iterator = { version = "0.2", default-features = false }
+fallible-iterator = { version = "0.2", default-features = false, optional = true }
 object = { version = "0.18", default-features = false, features = ["read"], optional = true }
-smallvec = { version = "1", default-features = false }
-lazycell = "1"
+smallvec = { version = "1", default-features = false, optional = true }
```

Comment:
Same for `smallvec`:

https://github.com/gimli-rs/gimli/blob/f346d6fe5434ae3fde26f4c1bf3ff31760690667/Cargo.toml#L39